transcom/mymove
# B-17796 Partial PPM Move Not Showing In SC's Closeout Queue

## Summary
For Partial PPM moves, it is not showing up in the Service Counselor's closeout queue.
This has to do with shipment status not changing correctly for the PPM shipment when the move has a HHG also.
The status for PPM shipment should change to "Waiting on Customer" once the Counselor finishes counseling.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create a new move with HHG and PPM shipments.
2. As a Service Counselor, review and finish counseling (Submit move details). (Make sure to check the right GBLOC)
3. The move should show up in SC's Closeout Tab. (Make sure to check the right GBLOC)

### Backend
